core/services/cable_guy/main.py

```diff
@@ -1,54 +1,68 @@
-#!/usr/bin/env python3
-import logging
+#! /usr/bin/env python3
+import json
 import os
 import sys
 from pathlib import Path
-from typing import Any
+from typing import Any, List
 
-import connexion
-import flask
-from connexion.resolver import RestyResolver
-from waitress import serve
+import uvicorn
+from api import manager
+from api.manager import EthernetInterface
+from fastapi import Body, FastAPI, HTTPException, status
+from fastapi.responses import HTMLResponse
+from fastapi.staticfiles import StaticFiles
+from starlette.responses import Response as StarletteResponse
+
+
+class PrettyJSONResponse(StarletteResponse):
+    media_type = "application/json"
+
+    def render(self, content: Any) -> bytes:
+        return json.dumps(
+            content,
+            ensure_ascii=False,
+            allow_nan=False,
+            indent=2,
+            separators=(", ", ": "),
+        ).encode(self.charset)
 
-logging.basicConfig(level=logging.INFO)
```

Comment:
Is there a reason to remove the log system ? This also does not match with the commit description.

Reply:
Uvicorn already activates logging at Info level by default.